frontend/src/scenes/insights/ActionFilter/ActionFilterRow/ActionFilterDropdown.tsx

```diff
@@ -183,7 +175,10 @@ export function ActionFilterDropdown({
     )
 }
 
-export function ActionInfo({ item }: { item: SelectedItem }): JSX.Element {
+export function ActionInfo({ item }: RenderInfoProps): JSX.Element {
```

Comment:
Nit: I don't think these need to be exported.

Reply:
`ActionInfo` is used in `SessionsFilterBox` for sessions filtering, but you're correct that the others don't need to be exported.